### DISCUSSION TITLE
post/openapi-documentation/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#45] [Announcements]
# post/openapi-documentation/

사실상의 표준으로 발돋움 중인 OpenAPI Specification을 이용한 API 문서화 방법(Swagger와 Spring REST Docs의 장점을 합치는 방법)을 공유드립니다.

https://tech.kakaopay.com/post/openapi-documentation/

**Comment by ACACIANh** [comment]
기회가 된다면 Spring REST Docs 적용해보고 싶습니다
좋은글 감사합니다

**Comment by ljw1126** [comment]
상세하게 잘 작성해주셔서 보기 좋네요 
잘 읽고 갑니다

**Comment by BGMSound** [comment]
안녕하세요! 글 잘 읽었습니다. 궁금한 점이 있어서 댓글 남깁니다! epages에서 제공하는 restdocs api spec이 RequestPart 등의 파라미터를 문서화하지 못하는 이슈가 있는 것으로 아는데 이를 어떤 식으로 해결하였나요?
